picard/album.py

```diff
@@ -610,12 +610,14 @@ def load(self, priority=False, refresh=False):
                 'artist-rels',
                 'recording-rels',
                 'release-rels',
+                'series-rels',
                 'url-rels',
                 'work-rels'
             }
             if config.setting['track_ars']:
                 inc |= {
                     'recording-level-rels',
+                    'release-group-level-rels',
```

Comment:
Hmm, I don't get why release group relationships should only be loaded when "Use track relationships" is enabled. Wouldn't it be more intuitive to add this `inc` parameter to the set which is enabled by "Use release relationships"?
(Would still not be 100% accurate as this is about release _group_ rels, but since "track" means recordings and works, it is also kind of intuitive to use "release" to also cover release groups.)

Reply:
Having this done with release ARs are enabled was actually my intention. I'll fix that.